### PR TITLE
INFRA: Initialize API Project

### LINKED ISSUE #3: INFRA: Initialize ASP.NET Core Web API Project
closes#2

### PR DESCRIPTION
**Pull request by NaynaMallick**
closes #3

### CODE REVIEW

File: ArtGallery.Services.Api/Program.cs [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace ArtGallery.Services.Api
+{
+    public class Program
+    {
+        public static void Main(string[] args)
```
Comment (ShriHumrudha) [inline ArtGallery.Services.Api/Program.cs]: This can be single line scope

File: ArtGallery.Services.Api/Startup.cs [hunk 2]
```diff
@@ -0,0 +1,57 @@
+using Microsoft.AspNetCore.Builder;
```
Comment (ShriHumrudha) [inline ArtGallery.Services.Api/Startup.cs:5]: We need copyrights header

File: ArtGallery.Services.Api/Startup.cs [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    public class Startup
+    {
+        public Startup(IConfiguration configuration) =>
+            Configuration = configuration;
+
+        public IConfiguration Configuration { get; }
+
+        public void ConfigureServices(IServiceCollection services)
+        {
+
```
Comment (ShriHumrudha) [inline ArtGallery.Services.Api/Startup.cs]: Remove empty line

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]

File: ArtGallery.Services.Api/Startup.cs (hunk 3 above)
Comment (NaynaMallick) [reply ArtGallery.Services.Api/Startup.cs]: removed

File: ArtGallery.Services.Api/Program.cs (hunk 1 above)
Comment (NaynaMallick) [reply ArtGallery.Services.Api/Program.cs]: changed

File: ArtGallery.Services.Api/Startup.cs (hunk 2 above)
Comment (NaynaMallick) [reply ArtGallery.Services.Api/Startup.cs:5]: added

File: .gitignore [hunk 4]
```diff
@@ -348,3 +348,4 @@ MigrationBackup/
 
 # Ionide (cross platform F# VS Code tools) working folder
 .ionide/
+/ArtGallery.Services.Api/appsettings.Development.json
```
Comment (ShriHumrudha) [inline .gitignore:351]: As we are ignoring the 'Development.json' please delete the file from this commit.

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]

**Review by Shrabanac — APPROVED** [review]

File: .gitignore (hunk 4 above)
Comment (NaynaMallick) [reply .gitignore:351]: deleted.

**Review by ShriHumrudha — APPROVED** [review]

**Pull request merged**